guides/common/modules/proc_cloning_satellite_server.adoc

```diff
@@ -161,7 +161,18 @@ If you copy to a different folder, update the `backup_dir` variable in the `/etc
 . Place the backup files from the source {Project} in the `/backup/` folder on the target server.
 You can either mount the shared storage or copy the backup files to the `/backup/` folder on the target server.
 . Power off the source server.
-. Enter the following commands to register to the Customer Portal, attach subscriptions, and enable only the required subscriptions:
+. Register the system to the Customer Portal, attach subscriptions, and enable only the required subscriptions:
```

Comment:
The subscriptions part is being resolved in https://github.com/theforeman/foreman-documentation/pull/3362.